packages/admin-ui-extensions-react/src/components/Heading/examples/Heading.example.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {extend, render, Heading} from '@shopify/admin-ui-extensions-react';
+
+function App() {
+  return <Heading>Heading</Heading>;
```

Comment:
Should the react/vanilla examples match?

Reply:
Absolutely! Just a miss on my part. :)